CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 * A config option and health checking for beginning support of emitting/receiving metrics via gRPC. Thanks, [eriwo-stripe](https://github.com/eriwo-stripe)!  
 * A gRPC server that listens for SSF spans and dogstatsd metrics on grpc_listening_addresses. Thanks [eriwo-stripe](https://github.com/eriwo-stripe) and [shrivu-stripe](https://github.com/shrivu-stripe)!
 * The ability to emit metrics from veneur-emit via the gRPC protocol as well as the option to specify a proxy for those metrics. Thanks [eriwo-stripe](https://github.com/eriwo-stripe) and [shrivu-stripe](https://github.com/shrivu-stripe)!
+* The "veneur.listening.received_per_protocol" metric to be published by global Veneur instances. This is a counter to track the metrics being directly listened for (rather than imported) by protocol. Thanks, [eriwo-stripe](https://github.com/eriwo-stripe)!
```

Comment:
Prefer `listen` to `listening` (in the same way that we prefer `flush` to `flushing`) and `received_per_protocol_total` (`_total` is convention so that we can easily identify the metric as a counter, similar to Hungarian notation) or `received_total`.

Reply:
Sounds good. Updating it to "veneur.listen.received_per_protocol_total".